spec/schemas/_core.bulk.yaml

```diff
@@ -29,20 +29,19 @@ components:
             dynamic_templates:
               description: |-
                 A map from the full name of fields to the name of dynamic templates.
-                Defaults to an empty map.
                 If a name matches a dynamic template, then that template will be applied regardless of other match predicates defined in the template.
                 If a field is already defined in the mapping, then this parameter won't be used.
               type: object
               additionalProperties:
                 type: string
             pipeline:
               description: |-
-                ID of the pipeline to use to preprocess incoming documents.
-                If the index has a default ingest pipeline specified, then setting the value to `_none` disables the default ingest pipeline for this request.
-                If a final pipeline is configured it will always run, regardless of the value of this parameter.
+                The pipeline ID for preprocessing documents.
+                When the index has a default ingest pipeline specified, then setting the value to `_none` disables the default ingest pipeline for this request.
+                When a final pipeline is configured, that pipeline will always run, regardless of the value of this parameter.
               type: string
             require_alias:
-              description: If `true`, the request's actions must target an index alias.
+              description: When `true`, require that all actions target an index alias rather than an index. Default is `false`.
```

Comment:
I think the schema supports `default: false`, when editing text you can add those too.